eclipse-collections-code-generator/src/main/resources/test/list/mutable/abstractPrimitiveListTestCase.stg

```diff
@@ -320,6 +320,24 @@ public abstract class Abstract<name>ListTestCase extends AbstractMutable<name>Co
         Assert.assertEquals(<(literal.(type))("1")>, list.get(0)<(wideDelta.(type))>);
     }
 
+    @Test
+    public void shuffleThis()
+    {
+        Mutable<name>List list = this.newWith(<["1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15"]:(literal.(type))(); separator=", ">);
+
+        Mutable<name>List checkList = this.newWith(<["1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15"]:(literal.(type))(); separator=", ">);
+
+        Mutable<name>List shuffleOne = this.newWith(list.shuffleThis().toArray());
```

Comment:
This could be replaced with:

<name>List shuffleOne = list.shuffleThis().toImmutable();
<name>List shuffleTwo = list.shuffleThis().toImmutable();

Reply:
It needs to be mutable because `sortThis()` is called on them later in the test.